### PR TITLE
POWR-1181 refactor group permissions

### PR DESCRIPTION
**Pull request by ccdatapdx**

### CODE REVIEW

**Comment by ccdatapdx** [comment]
I changed Marty Member to an "Editor" on the /powr group to pass the CI tests.

File: web/sites/default/config/group.role.elected_official-member.yml [hunk 1]
```diff
@@ -12,41 +12,7 @@ audience: member
 group_type: elected_official
 permissions_ui: true
 permissions:
-  - 'access group_media overview'
-  - 'access group_node overview'
```
Comment (kxwang) [inline web/sites/default/config/group.role.elected_official-member.yml]: Bureau member has "access group_node overview" and  "access group_media overview". Are they removed for Elected member on purpose?

File: web/sites/default/config/group.role.project-member.yml [hunk 2]
```diff
@@ -14,48 +14,17 @@ permissions_ui: true
 permissions:
   - 'access group_media overview'
   - 'access group_node overview'
-  - 'create group_media:audio entity'
-  - 'create group_media:document entity'
-  - 'create group_media:image entity'
-  - 'create group_media:video entity'
   - 'create group_node:city_service entity'
```
Comment (kxwang) [inline web/sites/default/config/group.role.project-member.yml]: Looks like members for Project can edit, which is different from members in other groups.

File: web/sites/default/config/group.role.elected_official-member.yml (hunk 1 above)
Comment (joshuami) [reply web/sites/default/config/group.role.elected_official-member.yml]: Nice catch. Fixed in the next commit.

File: web/sites/default/config/group.role.project-member.yml (hunk 2 above)
Comment (joshuami) [reply web/sites/default/config/group.role.project-member.yml]: This is weird. City Services is not enabled for the project group type. I can't explain why this permission would be here, but it doesn't do anything. I guess I could manually remove it.

Comment (joshuami) [reply web/sites/default/config/group.role.project-member.yml]: Manual removal seems to have no side effects.

**Pull request merged**